js/client-battle-tooltips.js

```diff
@@ -343,7 +348,21 @@ var BattleTooltips = (function () {
 				text += '&nbsp;SpA /&nbsp;' + myPokemon.stats['spd'] + '&nbsp;SpD /&nbsp;';
 			}
 			text += myPokemon.stats['spe'] + '&nbsp;Spe</p>';
-			if (isActive) text += '<p class="section">Opponent sees:</p>';
+			if (isActive) {
+				pokemon.stats = myPokemon.stats;
+				pokemon.ability = ability;
+				pokemon.item = item;
+				pokemon.hp = myPokemon.hp;
+				pokemon.maxhp = myPokemon.maxhp;
+				pokemon.unburden = myPokemon.unburden;
```

Comment:
Would that be better than what I'm currently doing? My thought process was that one slightly modified pokemon object would take up less memory than pokemon and a myPokemon would.